crates/api/src/fetch.rs

```diff
@@ -34,17 +48,35 @@ pub fn serialize_op_ids(value: Vec<OpId>) -> bytes::Bytes {
 /// Deserialize list of op ids.
 pub fn deserialize_op_ids(value: bytes::Bytes) -> K2Result<Vec<OpId>> {
     let op_ids = OpIds::decode(value).map_err(K2Error::other)?;
-    let vec = Vec::from(op_ids);
-    Ok(vec)
+    Ok(op_ids.into())
+}
+
+/// Serialize list of ops for sending over the wire.
+pub fn serialize_ops(value: Vec<MetaOp>) -> bytes::Bytes {
```

Comment:
Very similar to the previous op id one. I could write this as a generic function which saves us code here but requires then types when calling the function.